frontend/src/app/order-details/order-details.component.html

```diff
@@ -12,6 +12,6 @@
 </table>
 
 <button mat-raised-button *ngIf="order?.status == 'ASSEMBLED'" color="primary" (click)="openTrackingNr(order!)">Set tracking nr</button>
-<button mat-raised-button *ngIf="order?.status == 'CLAIMED'" color="primary" (click)="releaseOrder(order!.id!)" routerLink="/my-orders">Release order</button>
-<button mat-raised-button *ngIf="order?.status == 'CLAIMED'" color="primary" (click)="finishOrder(order!.id!)" routerLink="/my-orders">Finish order</button>
-<button mat-raised-button *ngIf="order?.status == 'NEW'" color="primary" (click)="claimOrder(order!.id!)" routerLink="/my-orders">Claim order</button>
+<button mat-raised-button *ngIf="order?.status == 'CLAIMED'" color="primary" (click)="releaseOrder(order!.id!)">Release order</button>
+<button mat-raised-button *ngIf="order?.status == 'CLAIMED'" color="primary" (click)="finishOrder(order!.id!)">Finish order</button>
```

Comment:
maybe the `CLAIMED` is copy paste error or these are intentionally?

Reply:
The assembler should be able to release the order and finish the order. So there will be two buttons available. And the initial status for these two functions is 'CLAIMED'.

Reply:
Actually you can see that in practice already. If you login to bonne.eu and claim an order, in 'my-orders' you will have  a choice to release or finish.